### PR TITLE
Use stylelint to lint CSS

### PR DESCRIPTION
**Pull request by cedric-anne**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

### CODE REVIEW

**Comment by cedric-anne** [comment]
This should be done using recommended rules and redefining only GLPI specific rules, like this: 

```js
{
  "extends": "stylelint-config-recommended"

  "indentation": 3
}
```

**Comment by cedric-anne** [comment]
Except for indentation rule, the other ones I override in the config file should be removed from here when we will refactor the UI. They are legitimate but requires too much work to be kept on current state of code.

File: css/jquery-glpi.css [hunk 1]
```diff
@@ -130,9 +130,6 @@ html .ui-autocomplete {
    border-width: 1px;
    border-style: solid;
    padding: .2em;
-}
-
-.ui-widget-content .error {
```
Comment (trasher) [inline css/jquery-glpi.css]: why this has been removed?

**Comment by trasher** [comment]
Could you please squash in one commit? Thanks ;)

File: css/jquery-glpi.css (hunk 1 above)
Comment (cedric-anne) [reply css/jquery-glpi.css]: It is the exact same selector as the previous one.

**Pull request merged**